jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/mapper/UserTaskInstanceQueryMapper.java

```diff
@@ -78,9 +78,7 @@ protected UserTaskInstanceDesc buildInstance(DataSet dataSetResult, int index) {
                 getColumnLongValue(dataSetResult, COLUMN_TASK_PROCESSINSTANCEID, index),//processInstanceId,
                 getColumnDateValue(dataSetResult, COLUMN_CREATEDON, index),//createdOn,
                 getColumnDateValue(dataSetResult, COLUMN_DUEDATE, index),//dueDate
-                getColumnLongValue(dataSetResult, COLUMN_WORKITEMID, index),//workItemId
-                getColumnDateValue(dataSetResult, COLUMN_TASK_SLA_DUE_DATE, index),//slaDueDate
-                getColumnIntValue(dataSetResult, COLUMN_TASK_SLA_COMPLIANCE, index)//slaCompliance
+                getColumnLongValue(dataSetResult, COLUMN_WORKITEMID, index)//workItemId
```

Comment:
Was this needed even before this change? This is only used for custom queries and since this mapper is aimed to be used only with "standard" UserTaskInstanceDesc fields which come from AuditTaskImpl I guess, it is OK. What if a user wants to obtain also SLA using custom queries? Then they would have to create their own mapper? I am fine with that, just asking.

Reply:
@MarianMacik the purpose of letting this workitemId is becasuse we need it to generate de second query in case the SLA data is requested.

Reply:
Yeah, I didn't question the workitemId, but the other SLA fields. So I guess when somebody uses this in a custom query, they can get SLA data in a separate query based on workitemId or even create a mapper like it was before, including the SLA fields, right?

Reply:
@MarianMacik  maybe I did it wrongly, but here I try to have at least the 'standard' mapping used by the getTaskById, but you are absolutely right they have to rely on their own mapper of the custom query. Would you suggest let this as it was then? I can remove also the workitemId. Makes sense?